tests/test_zz_jsonfield.py

```diff
@@ -17,7 +17,7 @@
     try:
         from django.db.models import JSONField as DjangoJSONField
     except ImportError:
-        from django.contrib.postgres.fields import JSONField as DjangoJSONField
+        from django.db.models import JSONField as DjangoJSONField
```

Comment:
this `try` / `except` can be removed now, and maybe the outer one, since `JSONField` is in Django 3.2+ for sure